iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Branch.java

```diff
@@ -57,16 +60,79 @@
 
     Side getSide(Terminal terminal);
 
-    CurrentLimits getCurrentLimits(Side side);
+    /**
+     * @deprecated Use {@link #getOperationalLimits1(LimitType, Class)} or {@link #getOperationalLimits2(LimitType, Class)} instead.
```

Comment:
I don't know if it's possible, but to have the same bahaviour you have to use the new method, but the first parameter is `LimitType.CURRENT` and the second one is `CurrentLimits.class`

Reply:
Add snipets of code in javadoc to specify it